pkgs/development/libraries/science/math/magma/generic.nix

```diff
@@ -113,13 +113,17 @@ stdenv.mkDerivation {
     lapack
     blas
   ] ++ lists.optionals cudaSupport (with cudaPackages; [
-    cuda_cudart
-    libcublas # cublas_v2.h
-    libcusparse # cusparse.h
+    cuda_cudart.dev # cuda_runtime.h
+    cuda_cudart.lib # cudart
+    cuda_cudart.static # cudart_static
+    libcublas.dev # cublas_v2.h
+    libcublas.lib # cublas
+    libcusparse.dev # cusparse.h
+    libcusparse.lib # cusparse
```

Comment:
IIRC there was an argument for using `lib.getDev pkg` instead of direct `pkg.dev`

Reply:
I think using the explicit attributes is better for our case because we get an error if they're missing. With the family of `get*` functions, we get a default if that specific output isn't found.

I strongly believe we should fail instead of producing an unintended configuration.